Cargo.toml

```
@@ -23,15 +23,15 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 serde_yaml = "0.9"
 snafu = "0.8"
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "stackable-operator-0.87.4" }
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "stackable-operator-0.87.5" }
 stackable-versioned = { git = "https://github.com/stackabletech/operator-rs.git", features = ["k8s"], tag = "stackable-versioned-0.7.0" }
 product-config = { git = "https://github.com/stackabletech/product-config.git", tag = "0.7.0" }
 strum = { version = "0.27", features = ["derive"] }
 tokio = { version = "1.40", features = ["full"] }
 tokio-zookeeper = "0.4"
 tracing = "0.1"
 
-#[patch."https://github.com/stackabletech/operator-rs"]
+# [patch."https://github.com/stackabletech/operator-rs"]
```

This space crept back in

```suggestion
#[patch."https://github.com/stackabletech/operator-rs"]
```